src/userConfig_sample.h

```diff
@@ -34,7 +34,7 @@ enum MACHINE {
 #define DISPLAYTEMPLATE 3          // 1 = Standard Display Template, 2 = Minimal Template, 3 = only Temperatur, 4 = Scale Template, 20 = vertical Display see git Handbook for further information
 #define DISPLAYROTATE U8G2_R0      // rotate display clockwise: U8G2_R0 = no rotation; U8G2_R1 = 90°; U8G2_R2 = 180°; U8G2_R3 = 270°
 #define SHOTTIMER 1                // 0 = deactivated, 1 = activated 2 = with scale
-#define HEATINGLOGO 0              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia
+#define HEATINGLOGO 3              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia, 3 = new Gaggia with new Heizlogo, 4 = Rancilio with new Heizlogo, 5 = Old Gaggia with new Heizlogo
```

Comment:
#define HEATINGLOGO 0              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia, 3 = new Gaggia with new heating logo, 4 = Rancilio with new heating logo, 5 = Old Gaggia with new heizlogo

gaggia and  new gaggia are two diffenrent mashines? if so, do they have clear names we can add?
define some with "new" is IMO a realy bad way. as you said you tryed to match the heating logo with the clevercoffee logo, so maybe we find way clear way to describe them. "generic heat logo" and clevercoffee heat logo".
Im open for any ideas here